samples/remote_attestation/common/attestation.h

```diff
@@ -31,7 +31,7 @@ class Attestation
      * the mrsigner and mrenclave values are tested to establish trust of the
```

Comment:
Aside: since this file is now mostly TEE-agnostic, it would be nice to use TEE agnostic terms instead of "mrsigner" and "mrenclave" in this comment.  Up to you whether you do anything in the PR or not, I'm ok either way.

Reply:
How about being consistent with OE claim naming?
mrsigner -> signer_id, mrenclave -> unique_id?

Reply:
That would be great, thanks!